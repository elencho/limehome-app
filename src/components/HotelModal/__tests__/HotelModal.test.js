import "react-native";
import React from "react";
import renderer from "react-test-renderer";

import  HotelModal from '../HotelModal'

it("renders correctly", () => {
  const tree = renderer.create(<HotelModal />).toJSON();
  expect(tree).toMatchSnapshot();
});
