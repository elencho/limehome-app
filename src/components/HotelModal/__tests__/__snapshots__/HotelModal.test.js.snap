// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Modal
  hardwareAccelerated={false}
  visible={true}
>
  <View
    style={
      Object {
        "backgroundColor": "#F7F1E9",
        "flex": 1,
        "justifyContent": "space-between",
      }
    }
  >
    <View>
      <View
        style={
          Object {
            "height": 600,
          }
        }
      >
        <RCTScrollView
          horizontal={true}
          pagingEnabled={true}
          showsHorizontalScrollIndicator={false}
        >
          <View />
        </RCTScrollView>
      </View>
      <View
        style={
          Object {
            "alignItems": "center",
            "flexDirection": "row",
            "justifyContent": "space-between",
            "marginHorizontal": 10,
            "marginTop": 20,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#4A4B4D",
              "fontSize": 19,
              "fontWeight": "bold",
            }
          }
        />
        <View
          style={
            Object {
              "alignItems": "center",
              "borderColor": "#4A4B4D",
              "borderRadius": 5,
              "borderWidth": 0.5,
              "flexDirection": "row",
              "padding": 5,
            }
          }
        >
          <Text
            style={
              Object {
                "color": "#4A4B4D",
                "fontSize": 19,
                "fontWeight": "bold",
              }
            }
          >
            4.5
          </Text>
          <Text />
        </View>
      </View>
      <View
        style={
          Object {
            "alignItems": "center",
            "flexDirection": "row",
            "marginHorizontal": 10,
          }
        }
      >
        <Text />
        <Text
          style={
            Object {
              "color": "#4A4B4D",
              "fontSize": 14,
              "fontWeight": "300",
            }
          }
        />
      </View>
      <Text
        style={
          Object {
            "color": "#4A4B4D",
            "fontSize": 17,
            "fontWeight": "400",
            "marginHorizontal": 10,
            "marginTop": 10,
          }
        }
      >
        Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.
      </Text>
      <View
        style={
          Object {
            "backgroundColor": "#4A4B4D",
            "height": 1,
            "marginHorizontal": 10,
            "marginVertical": 10,
          }
        }
      />
      <View
        style={
          Object {
            "marginHorizontal": 10,
          }
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "#4A4B4D",
                "fontSize": 19,
                "fontWeight": "bold",
              },
              Object {
                "marginVertical": 10,
              },
            ]
          }
        >
          Room Types available in this location
        </Text>
        <View
          style={
            Object {
              "flexDirection": "row",
              "flexWrap": "wrap",
            }
          }
        >
          <View
            style={
              Object {
                "alignSelf": "flex-start",
                "backgroundColor": "#C9CBAA",
                "borderRadius": 5,
                "marginBottom": 10,
                "marginRight": 10,
                "padding": 5,
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#4A4B4D",
                  "fontSize": 18,
                  "fontWeight": "400",
                }
              }
            >
              3x1 bedroom suites
            </Text>
          </View>
          <View
            style={
              Object {
                "alignSelf": "flex-start",
                "backgroundColor": "#C9CBAA",
                "borderRadius": 5,
                "marginBottom": 10,
                "marginRight": 10,
                "padding": 5,
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#4A4B4D",
                  "fontSize": 18,
                  "fontWeight": "400",
                }
              }
            >
              3x1 bedroom suites
            </Text>
          </View>
          <View
            style={
              Object {
                "alignSelf": "flex-start",
                "backgroundColor": "#C9CBAA",
                "borderRadius": 5,
                "marginBottom": 10,
                "marginRight": 10,
                "padding": 5,
              }
            }
          >
            <Text
              style={
                Object {
                  "color": "#4A4B4D",
                  "fontSize": 18,
                  "fontWeight": "400",
                }
              }
            >
              3x1 bedroom suites
            </Text>
          </View>
        </View>
      </View>
    </View>
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#C9CBAA",
          "flexDirection": "row",
          "justifyContent": "space-between",
          "paddingHorizontal": 15,
          "paddingVertical": 15,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#4A4B4D",
            "fontSize": 18,
            "fontWeight": "400",
          }
        }
      >
        From 
        <Text
          style={
            Object {
              "color": "#B16421",
            }
          }
        >
          15â‚¬
          /Night
        </Text>
      </Text>
      <View
        accessible={true}
        focusable={false}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#4C6447",
            "borderRadius": 5,
            "flexDirection": "row",
            "opacity": 1,
            "padding": 20,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#FFFFFF",
              "fontSize": 16,
              "fontWeight": "600",
            }
          }
        >
          EXPLORE 
        </Text>
        <Text />
      </View>
    </View>
  </View>
</Modal>
`;
